### PR TITLE
Add all Gemma-2b Configs

### PR DESCRIPTION
**Pull request by rasbt**
Adds all configs now, including tests. (I messed up the rebasing in the previous PR after we merged into main, hence this new PR)

### CODE REVIEW

File: config_hub/finetune/gemma-2b/full.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
+# The path to the base model's checkpoint directory to load for finetuning. (type: <class 'Path'>, default: checkpoints/stabilityai/stablelm-base-alpha-3b)
+checkpoint_dir: checkpoints/google/gemma-2b
+
+# Directory in which to save checkpoints and logs. (type: <class 'Path'>, default: out/lora)
+out_dir: out/finetune/google/full-gemma-2b
```
Comment (awaelchli) [inline config_hub/finetune/gemma-2b/full.yaml:6]: I saw you did it in the other files:
```suggestion
out_dir: out/finetune/full-gemma-2b
```

Comment (rasbt) [reply config_hub/finetune/gemma-2b/full.yaml:6]: Arg, that was on a separate studio and I must have forgotten to apply that update.

File: tests/test_config_hub.py [hunk 2]
```diff
@@ -8,23 +8,28 @@
...
+fixed_pairs = [
+    ("litgpt/pretrain.py", "pretrain/debug.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinyllama.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinystories.yaml"),
+    ("litgpt/pretrain.py", "https://raw.githubusercontent.com/Lightning-AI/litgpt/wip/config_hub/pretrain/tinystories.yaml"),
+]
+
+model_pairs = []
+models = ["gemma-2b", "llama-2-7b", "tiny-llama"]
+configs = ["full", "lora", "qlora"]
```
Comment (awaelchli) [inline tests/test_config_hub.py:20]: This looks good for now, but we might not always have all these combinations. 
In the future, we might want to replace this logic here by just reading the `config_hub` directory and listing each file, inferring the script name from the subfolder.

Comment (rasbt) [reply tests/test_config_hub.py]: That's a good point. I updated it to collect the configs automatically

File: tests/test_config_hub.py [hunk 3]
```diff
@@ -8,23 +8,31 @@
 from lightning.fabric.plugins import Precision
 
 
+fixed_pairs = [
+    ("litgpt/pretrain.py", "pretrain/debug.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinyllama.yaml"),
+    ("litgpt/pretrain.py", "pretrain/tinystories.yaml"),
+    ("litgpt/pretrain.py", "https://raw.githubusercontent.com/Lightning-AI/litgpt/wip/config_hub/pretrain/tinystories.yaml"),
```
Comment (carmocca) [inline tests/test_config_hub.py:15]:
```suggestion
    ("litgpt/pretrain.py", "https://raw.githubusercontent.com/Lightning-AI/litgpt/main/config_hub/pretrain/tinystories.yaml"),
```

**Review by carmocca — APPROVED** [review]

**Pull request merged**